realm/realm-library/src/androidTest/kotlin/io/realm/ParameterizedSetTests.kt

```diff
@@ -88,15 +89,15 @@ class ParameterizedSetTests(
         tester.contains()
     }
 
-    @Test
-    fun iterator() {
-        tester.iterator()
-    }
-
-    @Test
-    fun toArray() {
-        tester.toArray()
-    }
+//    @Test
```

Comment:
Using the pattern proposed in #7358 with annotations in the `Tester` and inheriting that, would make it easier to comment tests in and out :)

Reply:
I will address this in https://github.com/realm/realm-java/issues/7362